src/parser/csql_grammar.y

```diff
@@ -4020,6 +4091,35 @@ rename_stmt
                             node->info.rename_server.new_name = $5;
 			  }
 
+			$$ = node;
+			PARSER_SAVE_ERR_CONTEXT ($$, @$.buffer_pos)
+		DBG_PRINT}}
+	| RENAME			/* 1 */
+		{ push_msg (MSGCAT_SYNTAX_SYNONYM_INVALID_RENAME); }	/* 2 */
+	  opt_access_modifier		/* 3 */
+	  SYNONYM			/* 4 */
+	  synonym_name			/* 5 */
+	  as_or_to			/* 6 */
+	  synonym_name_without_dot	/* 7 */
+	  	{ pop_msg(); }
+		{{
+
```

Comment:
Please add below
DBG_TRACE_GRAMMAR(rename_stmt, | RENAME opt_access_modifier SYNONYM synonym_name as_or_to synonym_name_without_dot);

Reply:
Thank you. Added.